### PR TITLE
fix(DB/Quest): gluttonous lurkers: improve the current SmartAI

### PR DESCRIPTION
**Pull request by Helias**
Thanks @r0m1ntik94 for the fix.

## CHANGES PROPOSED
- improve the current SmartAI

## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3227

## TESTS PERFORMED:
- executed successfully the SQL query into the database

## HOW TO TEST THE CHANGES:
- complete the quest https://www.wowhead.com/quest=12713/betrayal

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1598303195521812924.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1598303195521812924');
+DELETE FROM `smart_scripts`WHERE `entryorguid`= 28202;
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1598303195521812924.sql:2]: this `DELETE` is bad! they primary key of `smart_scripts` is `entryorguid` AND `source_type`. If you don't specify the `source_type`, you can accidentally remove other scripts. Be careful when using `DELETE`

**Review by FrancescoBorzi — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1598303195521812924.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+INSERT INTO `smart_scripts` VALUES
+(28202, 0, 0, 0, 8, 0, 100, 1, 50926, 0, 0, 0, 0, 11, 50927, 3, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Zul\'Drak Rat - On Spellhit \'Gluttonous Lurkers: Create Zul\'Drak Rat Cover\' - Cast `Create Zul\'Drak Rat`'),
+(28202, 0, 1, 0, 31, 0, 100, 1, 50927, 0, 0, 0, 0, 41, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Zul\'Drak Rat - On Spellhit target \'Create Zul\'Drak Rat\' - Despawn Instant');
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1598303195521812924.sql:5]: newline

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by PivanDepolo** [comment]
works fine!

File: data/sql/updates/pending_db_world/rev_1598303195521812924.sql [hunk 3]
```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1598303195521812924');
+DELETE FROM `smart_scripts` WHERE `entryorguid`= 28202 AND `source_type` = 0;
+INSERT INTO `smart_scripts` VALUES
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1598303195521812924.sql:3]: Missing (`Entryorguid, [....] VALUES

Comment (Helias) [reply data/sql/updates/pending_db_world/rev_1598303195521812924.sql]: added!

**Pull request merged**